WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/HelpAndSupportViewController.swift

```diff
@@ -229,6 +303,16 @@ private extension HelpAndSupportViewController {
         ZendeskManager.shared.showNewRequestIfPossible(from: navController)
     }
 
+    /// Contact WCPay Support action
+    ///
+    func contactWCPaySupportWasPressed() {
```

Comment:
How about `contactPaymentSupportWasPressed`?

Reply:
I also think we should be specific here, at least for now. This is the action that submits a ticket that is tagged precisely to go into the WCPay queue.